Submodule CMakeLists.txt changes aren't detected by find-changed-files

UMD bump https://github.com/tenstorrent/tt-metal/pull/31203 contains a change https://github.com/tenstorrent/tt-umd/pull/1517 that bumped the required cmake version from 3.24 to 3.27

This caused issues in tt-metal when running PR and merge gate after the change
```
CMake Error at tt_metal/third_party/umd/CMakeLists.txt:1 (cmake_minimum_required):
  CMake 3.27 or higher is required.  You are running version 3.24.4
```

find-changed-files wasn't able to see that CMakeLists.txt in UMD was changed, and as a result the check-cmake-version check didn't run
https://github.com/tenstorrent/tt-metal/actions/runs/18834598947/job/53732613205

--- a/.github/scripts/utils/find-changed-files.sh
+++ b/.github/scripts/utils/find-changed-files.sh
@@ -88,8 +88,8 @@ if [[ "$SUBMODULE_CHANGED" = true ]]; then
     # Something to make more efficient in future.
     TOOLS_CHANGED=true
     ANY_CODE_CHANGED=true
-	# Issue: https://github.com/tenstorrent/tt-metal/issues/31344
-	CMAKE_CHANGED=true
+    # Issue: https://github.com/tenstorrent/tt-metal/issues/31344
+    CMAKE_CHANGED=true
 fi
 
 declare -A changes=(

--- a/.github/scripts/utils/find-changed-files.sh
+++ b/.github/scripts/utils/find-changed-files.sh
@@ -88,6 +88,8 @@ if [[ "$SUBMODULE_CHANGED" = true ]]; then
     # Something to make more efficient in future.
     TOOLS_CHANGED=true
     ANY_CODE_CHANGED=true
+	# Issue: https://github.com/tenstorrent/tt-metal/issues/31344
+	CMAKE_CHANGED=true
 fi
 
 declare -A changes=(

